.github/workflows/integration.yml

```diff
@@ -141,7 +141,7 @@ jobs:
       - uses: ./agoric-sdk/.github/actions/restore-node
         with:
           # XXX loadgen not compatible with 18.19 https://github.com/Agoric/agoric-sdk/pull/8365#issuecomment-1848003597
-          node-version: 18.18
+          version-identifier: 18.18
```

Comment:
Hopefully we can move up soon when we remove the loadgen test from `deployment-test`